userguide/tutorials/debugging.adoc

```diff
@@ -1,5 +1,47 @@
 = Debugging tips
 
+== Finding the current Kill Bill version
+
+You can find the current Kill Bill version by invoking the following endpoint:
```

Comment:
How about adding something here about the plugin information that's also displayed. (Assuming that's true based on the analytics info I see in the sample.)